staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go

```diff
@@ -234,7 +234,7 @@ func (o *CertificateOptions) modifyCertificateCondition(builder *resource.Builde
 			csr := info.Object.(*certificatesv1beta1.CertificateSigningRequest)
 			csr, hasCondition := modify(csr)
```

Comment:
This can also be `_, hasCondition`

Reply:
Seems like that, but I prefer to keep it like that for the design of the modify function. Some one may construct a totally new `certificatesv1beta1.CertificateSigningRequest` object :thinking: